website/docs/r/app_service_managed_certificate.html.markdown

```diff
@@ -0,0 +1,114 @@
+---
+subcategory: "App Service (Web Apps)"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_managed_certificate"
+description: |-
+  Manages an App Service Managed Certificate.
+---
+
+# azurerm_app_service_managed_certificate
+
+Manages a App Service Managed Certificate.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "exampleRG"
+  location = "West Europe"
+}
+
+resource "azurerm_app_service_plan" "example" {
+  name                = "examplePlan"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  kind                = "Linux"
+  reserved            = true
+
+  sku {
+    tier = "Basic"
+    size = "B1"
+  }
+}
+
+resource "azurerm_app_service" "example" {
+  name                = "exampleApp"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  app_service_plan_id = azurerm_app_service_plan.example.id
+}
+
+resource "azurerm_dns_txt_record" "test" {
+  name                = join(".", ["asuid", "%s"])
+  zone_name           = data.azurerm_dns_zone.test.name
+  resource_group_name = data.azurerm_dns_zone.test.resource_group_name
+  ttl                 = 300
+
+  record {
+    value = azurerm_app_service.test.custom_domain_verification_id
+  }
+}
+
+resource "azurerm_app_service_custom_hostname_binding" "example" {
+  hostname            = "mycustomhost.contoso.com"
+  app_service_name    = azurerm_app_service.example.name
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+
+resource "azurerm_app_service_managed_certificate" "example" {
+  custom_hostname_binding_id = "TODO"
```

Comment:
TODO?

Reply:
Would appear I missed it from the docs generator 🙈